style.css

```diff
@@ -1,10 +1,29 @@
+html,
+body {
+  width: 100vw;
+  height: 100vh;
+  overflow: hidden;
+  background-color: black;
+  margin: 0;
+}
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
I don't know why there has to be two body declarations here; I just know that aMoniker did that in Gush, and I'm too scared to change it.

Also vw and vh refer to percentage of the viewable width and viewable height respectively; so 100vw is 100% of the viewable width.

Reply:
You don't need 2 body declarations, it was probably for visual separation of different types of styling